### PR TITLE
Another small `MockWebSocket` improvement

### PR DESCRIPTION
**Pull request by nvie**
Another small improvement for working with the new `MockWebSocketServer` abstraction in unit tests. Since in all cases so far, we're dealing with JSON messages, observing the list of received messages should just work with JSON. In the 0.01% case, where we may want to write a test for non-JSON data (probably to test for error handling, or `ping` messages), we can fall back to the raw one.

Saves a _lot_ of `JSON.stringify`'s, and also makes test failure output more readable.

### CODE REVIEW

File: packages/liveblocks-core/src/__tests__/_utils.ts [hunk 1]
```diff
@@ -123,6 +123,21 @@ export class MockWebSocketServer {
...
+   * In 99.9% of cases, the server messages are going to be JSON-serialized
+   * values. If you need to look at the raw message (the 0.1% case), you can
+   * use this.receivedMessagesRaw for that.
+   */
+  public get receivedMessages(): Json[] {
+    return this.receivedMessagesRaw.map((raw) => {
+      try {
+        return JSON.parse(raw) as Json;
+      } catch {
+        return "<non-JSON value>";
```
[the remark this answers is not in the document]
Comment (nvie) [reply packages/liveblocks-core/src/__tests__/_utils.ts:136]: My only worry with it would be that `raw = 'ping'` would not be valid JSON, but `raw = '"ping"'` would, and I thought that could be confusing if that'd happen implicitly. I used this explicit string value to recognize this edge case, rather than hide it. That said, I haven't seen this code path taken just yet in all the tests I've rewritten so far, so perhaps we could even throw in this case.
